Tests/UnitTests/Views/TableViewTests.cs

```diff
@@ -3387,6 +3387,48 @@ public void TableCollectionNavigator_FullRowSelect_True_False (bool fullRowSelec
         Assert.Equal (expectedRow, tableView.CollectionNavigator.GetNextMatchingItem (0, "3".ToCharArray () [0]));
     }
 
+
```

Comment:
If you are not testing HotKeyBindings, and just KeyBindings, I don' think there's a need to depend on Application.Top in this case. 

It's good practice when writing unit tests to strive to simplify dependencies as much as possible and try to make them work as a parallelizable test. I do recognize that KeyBindings tends to put a wrench in that sometimes though.

Reply:
Yes, like that. 

- Strive to build tests that don't depend on `Application` or `ConfigurationManager`.
- Put those tests that don't depend on `Application` or `ConfigurationManager` in `UnitTestsParallezable`.

The following have implicit/non-obvious dependencies on `Application`:

- `HotKeyBindings`
- Complex Navigation - Anytime, `Application.Navigation.GetFocused ()` may come into play.

Of course, once the `Applicationv2` stuff is fully done we can leverage `ChangeInstance()` to make even tests that depend on `Application` parallelizable! (Thank you!!!)